### PR TITLE
replace graphiql references

### PR DESCRIPTION
**Pull request by mirnawong1**
replacing any graphiql references with graphql explorer, per [slack thread](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1679408421765959) from @b-per  and @cafzal 

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Comment by b-per** [comment]
Not sure if the wording needs to be tweaked just a bit.

GraphiQL used to refer to only the interactive interface to GraphQL, and it had a different endpoint from the "normal" API endpoint.

Now, I am just worried that there might be a bit of confusion between GraphQL (the API format) which has the endpoint https://metadata.{YOUR_ACCESS_URL}/graphql and the GraphQL Explorer (the interactive website), which has the same endpoint as the API itself.

It might be OK to always refer to the interactive part as the **GraphQL Explorer** maybe?

**Comment by b-per** [comment]
Also, we might want to update the screenshot with a query that provides a correct answer rather than BAD_USER_INPUT 😄

**Comment by mirnawong1** [comment]
Oh ok I see so I've confused the two for sure. So the ##GraphQL section in the **Query** page - is the below correct?

## GraphQL

Run queries directly in the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and use the document explorer on the left-hand side, where you can see all possible nodes and fields. 

The following is an example of what a query looks like in GraphQL: 

<Lightbox src="/img/docs/dbt-cloud/metadata-api/graphql.jpg" width="85%" title="Run queries using the Apollo Server GraphQL explorer"/>

> Not sure if the wording needs to be tweaked just a bit.
> 
> GraphiQL used to refer to only the interactive interface to GraphQL, and it had a different endpoint from the "normal" API endpoint.
> 
> Now, I am just worried that there might be a bit of confusion between GraphQL (the API format) which has the endpoint [https://metadata.{YOUR_ACCESS_URL}/graphql](https://metadata.%7BYOUR_ACCESS_URL%7D/graphql) and the GraphQL Explorer (the interactive website), which has the same endpoint as the API itself.
> 
> It might be OK to always refer to the interactive part as the **GraphQL Explorer** maybe?

File: website/docs/docs/dbt-cloud-apis/metadata-querying.md [hunk 1]
```diff
@@ -32,9 +33,18 @@ Every query will rely on a _jobID_.  You can get the jobID by clicking into the
...
-You can experiment and run queries directly in the [GraphiQL interface](https://metadata.cloud.getdbt.com/graphiql), making sure to use the [appropriate Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan. On the right-hand side, there is a document explorer where you can see all possible nodes and fields.  Below is an example of what a query looks like in GraphiQL.  Note that you must authenticate via bearer auth with your token.
 
-<Lightbox src="/img/docs/dbt-cloud/GraphiQL.png" title=""/>
+## GraphQL API explorer
+
+You can run ad-hoc queries directly in the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and use the document explorer on the left-hand side, where you can see all possible nodes and fields. 
+
+1. Access the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and select fields you'd like query
+2. Go to **Variables** at the bottom of the explorer and replace any `null` fields with your unique fields
+3. To [authenticate](#run-queries) via bearer auth with `YOUR_TOKEN`, go to **Headers** at the bottom of the explorer and select **+New header**
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/metadata-querying.md]:
```suggestion
3. To authenticate via bearer auth with `YOUR_TOKEN`, go to **Headers** at the bottom of the explorer and select **+New header** 
```

File: website/docs/docs/dbt-cloud-apis/metadata-querying.md [hunk 2]
```diff
@@ -32,9 +33,18 @@ Every query will rely on a _jobID_.  You can get the jobID by clicking into the
...
-## GraphiQL
 
-You can experiment and run queries directly in the [GraphiQL interface](https://metadata.cloud.getdbt.com/graphiql), making sure to use the [appropriate Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan. On the right-hand side, there is a document explorer where you can see all possible nodes and fields.  Below is an example of what a query looks like in GraphiQL.  Note that you must authenticate via bearer auth with your token.
 
-<Lightbox src="/img/docs/dbt-cloud/GraphiQL.png" title=""/>
+## GraphQL API explorer
+
+You can run ad-hoc queries directly in the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and use the document explorer on the left-hand side, where you can see all possible nodes and fields. 
+
+1. Access the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and select fields you'd like query
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/metadata-querying.md]:
```suggestion
1. Access the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and select fields you'd like query. Refer to the [Apollo explorer](https://www.apollographql.com/docs/graphos/explorer/explorer) for setup and authorization info. 
```

File: website/docs/docs/dbt-cloud-apis/metadata-querying.md [hunk 3]
```diff
@@ -32,9 +33,18 @@ Every query will rely on a _jobID_.  You can get the jobID by clicking into the
...
-## GraphiQL
 
-You can experiment and run queries directly in the [GraphiQL interface](https://metadata.cloud.getdbt.com/graphiql), making sure to use the [appropriate Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan. On the right-hand side, there is a document explorer where you can see all possible nodes and fields.  Below is an example of what a query looks like in GraphiQL.  Note that you must authenticate via bearer auth with your token.
 
-<Lightbox src="/img/docs/dbt-cloud/GraphiQL.png" title=""/>
+## GraphQL API explorer
+
+You can run ad-hoc queries directly in the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and use the document explorer on the left-hand side, where you can see all possible nodes and fields. 
+
+1. Access the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and select fields you'd like query. Refer to the [Apollo explorer](https://www.apollographql.com/docs/graphos/explorer/explorer) for setup and authorization info.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/metadata-querying.md]:
```suggestion
1. Access the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and select fields you'd like query. Refer to the [Apollo explorer documentation](https://www.apollographql.com/docs/graphos/explorer/explorer) for setup and authorization info. 
```

File: website/docs/docs/dbt-cloud-apis/metadata-querying.md [hunk 4]
```diff
@@ -32,9 +33,18 @@ Every query will rely on a _jobID_.  You can get the jobID by clicking into the
...
-<Lightbox src="/img/docs/dbt-cloud/GraphiQL.png" title=""/>
+## GraphQL API explorer
+
+You can run ad-hoc queries directly in the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and use the document explorer on the left-hand side, where you can see all possible nodes and fields. 
+
+1. Access the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and select fields you'd like query. Refer to the [Apollo explorer documentation](https://www.apollographql.com/docs/graphos/explorer/explorer) for setup and authorization info. 
+2. Go to **Variables** at the bottom of the explorer and replace any `null` fields with your unique fields
+3. To authenticate via bearer auth with `YOUR_TOKEN`, go to **Headers** at the bottom of the explorer and select **+New header** 
+4. Select **Authorization** in the **header key** drop-down list and enter your Bearer auth token in the **value** field 
+<Lightbox src="/img/docs/dbt-cloud/metadata-api/graphql_header.jpg" width="85%" title="Add your Bearer auth token in the Apollo Server GraphQL explorer"/>
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/metadata-querying.md]:
```suggestion

```

File: website/docs/docs/dbt-cloud-apis/metadata-querying.md [hunk 5]
```diff
@@ -32,9 +33,17 @@ Every query will rely on a _jobID_.  You can get the jobID by clicking into the
...
 
-<Lightbox src="/img/docs/dbt-cloud/GraphiQL.png" title=""/>
+## GraphQL API explorer
+
+You can run ad-hoc queries directly in the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and use the document explorer on the left-hand side, where you can see all possible nodes and fields. 
+
+1. Access the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and select fields you'd like query. Refer to the [Apollo explorer documentation](https://www.apollographql.com/docs/graphos/explorer/explorer) for setup and authorization info. 
+2. Go to **Variables** at the bottom of the explorer and replace any `null` fields with your unique fields
+3. To authenticate via bearer auth with `YOUR_TOKEN`, go to **Headers** at the bottom of the explorer and select **+New header**
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/metadata-querying.md]:
```suggestion
3. To [authenticate](https://www.apollographql.com/docs/graphos/explorer/connecting-authenticating#authentication) via bearer auth with `YOUR_TOKEN`, go to **Headers** at the bottom of the explorer and select **+New header** 
```

File: website/docs/docs/dbt-cloud-apis/metadata-querying.md [hunk 6]
```diff
@@ -32,9 +33,17 @@ Every query will rely on a _jobID_.  You can get the jobID by clicking into the
...
-## GraphiQL
 
-You can experiment and run queries directly in the [GraphiQL interface](https://metadata.cloud.getdbt.com/graphiql), making sure to use the [appropriate Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan. On the right-hand side, there is a document explorer where you can see all possible nodes and fields.  Below is an example of what a query looks like in GraphiQL.  Note that you must authenticate via bearer auth with your token.
 
-<Lightbox src="/img/docs/dbt-cloud/GraphiQL.png" title=""/>
+## GraphQL API explorer
+
+You can run ad-hoc queries directly in the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and use the document explorer on the left-hand side, where you can see all possible nodes and fields. 
+
+1. Access the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and select fields you'd like query. Refer to the [Apollo explorer documentation](https://www.apollographql.com/docs/graphos/explorer/explorer) for setup and authorization info.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/metadata-querying.md]:
```suggestion
1. Access the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and select fields you'd like query. 
```

File: website/docs/docs/dbt-cloud-apis/metadata-querying.md [hunk 7]
```diff
@@ -32,9 +33,17 @@ Every query will rely on a _jobID_.  You can get the jobID by clicking into the
...
 There are several illustrative example queries in this documentation. You can see an example of [queries on the Model node](/docs/dbt-cloud-apis/metadata-schema-model).
 
-## GraphiQL
 
-You can experiment and run queries directly in the [GraphiQL interface](https://metadata.cloud.getdbt.com/graphiql), making sure to use the [appropriate Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan. On the right-hand side, there is a document explorer where you can see all possible nodes and fields.  Below is an example of what a query looks like in GraphiQL.  Note that you must authenticate via bearer auth with your token.
 
-<Lightbox src="/img/docs/dbt-cloud/GraphiQL.png" title=""/>
+## GraphQL API explorer
+
+You can run ad-hoc queries directly in the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and use the document explorer on the left-hand side, where you can see all possible nodes and fields.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/metadata-querying.md:40]:
```suggestion
You can run ad-hoc queries directly in the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and use the document explorer on the left-hand side, where you can see all possible nodes and fields. 

Refer to the [Apollo explorer documentation](https://www.apollographql.com/docs/graphos/explorer/explorer) for setup and authorization info. 
```

**Comment by mirnawong1** [comment]
thank you again for teaching me @b-per - i've made changes based on our chat. let me know what you think 🙏 

https://deploy-preview-3047--docs-getdbt-com.netlify.app/docs/dbt-cloud-apis/metadata-querying

**Review by b-per — APPROVED** [review]
Thanks Mirna, I feel like it is quite clearer now!

**Comment by mirnawong1** [comment]
thank you so much for your guidance here Benoit! MVP 🙏

**Pull request merged**